include/wx/persist/dataview.h

```diff
@@ -0,0 +1,169 @@
+///////////////////////////////////////////////////////////////////////////////
+// Name:        wx/persist/dataview.h
+// Purpose:     Persistence support for wxDataViewListCtrl
+// Author:      wxWidgets Team
+// Created:     2017-08-21
+// Copyright:   (c) 2017 wxWidgets.org
+// Licence:     wxWindows licence
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef _WX_PERSIST_DATAVIEW_H_
+#define _WX_PERSIST_DATAVIEW_H_
+
+#include "wx/persist/window.h"
+#include "wx/dataview.h"
+
+// ----------------------------------------------------------------------------
+// String constants used by wxPersistentDataViewListCtrl.
```

Comment:
Sorry if I am wrong but it seems there are still remaining texts in this file containing "DataView**List**Ctrl": Should not these be replaced with just "DataViewCtrl" everywhere now? Perhaps similarly with "DVLC" -> "DV" in the wxPERSIST_* #defines...